package.json

```diff
@@ -18,6 +18,7 @@
     "react": "^16.8.3",
     "react-dom": "^16.8.3",
     "react-google-maps": "^9.4.5",
+    "react-media": "^1.9.2",
```

Comment:
I intended to use it a while ago but gave up later. It would be useful when we have different HTML components for different platforms. But now we don't.